nix-channels.py

```diff
@@ -372,6 +372,9 @@ def garbage_collect():
     alive = set()
 
     for release in (working_dir / RELEASES_DIR).iterdir():
+        # This release never finished downloading
+        if (release / 'binary-cache-url').exists(): continue
```

Comment:
It should be `if not (release / 'binary-cache-url').exists(): continue` or EVERY package in `store/` will be deleted (which happens in USTC Mirrors last night after cherry-picking this commit).

See https://github.com/ustclug/ustcmirror-images/commit/bf4c3e17cc4324114aa71a416e96b60b01bbd437

Reply:
Indeed, I'm very sorry about that....